Automattic/wp-calypso
# My Home: Update illustration for connect accounts task

#### Changes proposed in this Pull Request

* Update the illustration used for the "Connect accounts" task on the My Home experimental layout.
* Tweak a few typography styles for on the task card for better balance at medium breakpoints.

Before | After
------------ | -------------
<img width="1079" alt="Screen Shot 2020-04-24 at 12 15 24 PM" src="https://user-images.githubusercontent.com/448298/80234068-80bcf300-8625-11ea-8b86-0290639451c6.png"> | <img width="1087" alt="Screen Shot 2020-04-24 at 12 11 49 PM" src="https://user-images.githubusercontent.com/448298/80234063-7d296c00-8625-11ea-938d-f932dafeda40.png">
![Screen Shot 2020-04-24 at 12 16 15](https://user-images.githubusercontent.com/448298/80234133-9c27fe00-8625-11ea-8969-248323857480.png) | ![Screen Shot 2020-04-24 at 12 16 32](https://user-images.githubusercontent.com/448298/80234117-96cab380-8625-11ea-98a7-58e249436846.png)


#### Testing instructions



* Apply D42273-code and sandbox the API.
* Using an established site that doesn't have any Publicize connections: Load the experimental Home layout by going to `/home/:site?flags=home/experimental-layout`.
* Make sure the new illustration shows up on the "Connect accounts" task card at the top.
* Change your viewport size to 800px.
* Reload the page.
* Make sure the font sizes look ok and the illustration is not showing.
